dist/rpm/openQA.spec

```diff
@@ -388,6 +388,10 @@ ln -s %{_datadir}/openqa/script/openqa-clone-job %{buildroot}%{_bindir}/openqa-c
 ln -s %{_datadir}/openqa/script/openqa-dump-templates %{buildroot}%{_bindir}/openqa-dump-templates
 ln -s %{_datadir}/openqa/script/openqa-load-templates %{buildroot}%{_bindir}/openqa-load-templates
 ln -s %{_datadir}/openqa/script/openqa-clone-custom-git-refspec %{buildroot}%{_bindir}/openqa-clone-custom-git-refspec
+ln -s %{_datadir}/openqa/script/occ %{buildroot}%{_bindir}/occ
+ln -s %{_datadir}/openqa/script/ocj %{buildroot}%{_bindir}/ocj
+ln -s %{_datadir}/openqa/script/ocl %{buildroot}%{_bindir}/ocl
+ln -s %{_datadir}/openqa/script/ola %{buildroot}%{_bindir}/ola
```

Comment:
I wonder if these aren't more suited to aliases or at least links setup by users - no super strong opinion but if these also end up in shared scripts or config files it might make it more difficult to follow the code or check where commands are used